cli/azd/pkg/keyvault/keyvault.go

```diff
@@ -356,27 +356,45 @@ func NewAkvs(subId, vaultId, secretName string) string {
 }
 
 func (kvs *keyVaultService) SecretFromAkvs(ctx context.Context, akvs string) (string, error) {
+	parseAkvs, err := ParseAkvs(akvs)
+	if err != nil {
+		return "", err
+	}
+	subscriptionId, vaultName, secretName := parseAkvs.SubscriptionId, parseAkvs.VaultName, parseAkvs.SecretName
```

Comment:
This is a cool little variable assignment notation that I wasn't aware of.  Is there a reason to create these new variables vs using the parseAkvs.* members in the next call to get the keyvault secret?